pytest/batching/test_batch.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+import torch
+from ml_gym.batching.batch import InferenceResultBatch, DatasetBatch
+
+
+class TestInferenceResultBatch:
+    target_key = "target_key"
+    prediction_key = "prediction_key"
+
+    @pytest.fixture
+    def inference_batch_result(self) -> InferenceResultBatch:
+        targets = torch.IntTensor([0, 0, 0, 1, 1, 1])
+        tags = torch.IntTensor([0, 0, 0, 1, 1, 1])
+        predictions = torch.FloatTensor([0, 0.1, 0.01, 0.9, 0.7, 1])
+        return InferenceResultBatch(targets={TestInferenceResultBatch.target_key: targets},
+                                    predictions={TestInferenceResultBatch.prediction_key: predictions},
+                                    tags=tags)
+
+    # test device operations
```

Comment:
Without individual assertions, those tests are a bit redundant or?

Reply:
yeah they run "at least" through then. In the deployment environment we unfortunately don't have any GPUs to test with.